nettacker/database/db.py

```
@@ -94,17 +138,47 @@ def submit_report_to_db(event):
+                INSERT INTO reports (date, scan_unique_id, report_path_filename, options)
+                VALUES (?, ?, ?, ?)
+                """,
+                (
+                    str(event["date"]),
+                    event["scan_id"],
+                    event["options"]["report_path_filename"],
+                    json.dumps(event["options"]),
+                ),
+            )
+            return send_submit_query(session)
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)